docs/use_cases.md

```diff
@@ -0,0 +1,74 @@
+# Node.js Use Cases
+
+This doc is intended to capture the key uses cases for Nodej.js. These
+use cases will then be used to guide what we develop/run as part of
+the community benchmarking effort.
+
+# Main Use Case Categories
+
+The folloing categories have been identifed:
+
+* REST or REST like back end services
+* Generating/serving dynamic web page content
+* AJAX like applications with bidirectional communication over
+  websockets and/or http2
+* APIs/microservices based applications
+* Mobile back ends
+* Small scripts
+
+For these use cases one or more of the following are often imporant:
+
+* consistent low latency
+* ability to support high concurrency
+* throughput
+* fast startup/restart/shutdown
+* low resource usage (memory/cpu)
+
+## REST or REST like back end services
```

Comment:
title as per above comment, same goes for the other titles, sorry for being so picky, I can't help it!